packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/upgrade/translations/Messages_fr_FR.json

```
@@ -3,6 +3,7 @@
   "dedicatedCloud_datacenter_backup_upgrade_confirm": "Passer à l’offre {{ name }}",
   "dedicatedCloud_datacenter_backup_upgrade_cancel": "Annuler",
   "dedicatedCloud_datacenter_backup_upgrade_description": "Vous souhaitez changer votre offre de sauvegarde de l’<strong>offre {{ currentOffer }}</strong> à l’<strong>offre {{ selectedOffer }}</strong>.",
+  "dedicatedCloud_datacenter_backup_upgrade_warning": "Si vous avez activé la sauvegarde sur d’autres datacentres de ce PCC, cette offre leur sera également appliquée.",
```

Maybe ? If you're speaking of Private Cloud in other translations
```suggestion
  "dedicatedCloud_datacenter_backup_upgrade_warning": "Si vous avez activé la sauvegarde sur d’autres datacentres de ce Private Cloud, cette offre leur sera également appliquée.",
```

---

This is changed :)